slider.css

```diff
@@ -0,0 +1,57 @@
+.slider {
+  display: block;
+}
+
+.slider-container {
+  display: inline-block;
+  position: relative;
+}
+
+.slider-list {
+  padding: 0;
+  margin: 0;
+}
+
+.slider-item {
+  display: none;
+}
+
+.slider-item-active {
+  display: block;
+}
+
+.navigation-buttons {
+  display: none;
+}
+
+.navigation-buttons .btn {
+  position: absolute;
+  top: 50%;
+  transform: translate(0, -50%);
+}
+
+.navigation-buttons .btn.btn-prev {
```

Comment:
It's bad approach of making such cascades. When you need to override behaviour, it will cause more difficult selectors. If this button is specific rather to the slider than to the entire theme, it should have a specific class. 

.slider-navigation for navigation-buttons
.slider-navigation-button for buttons inside
.slider-navigation-button-prev - to add left: 0;